docs/iris/src/whatsnew/aggregate_directory.py

```diff
@@ -176,9 +260,16 @@ def run_compilation(release):
         for string_line in compiled_text:
             output_object.write(string_line)
 
+
 if __name__ == '__main__':
     PARSER = argparse.ArgumentParser()
+    _DUMMY_VERSION = '9999.9999'
     PARSER.add_argument("release", help="Release number to be compiled",
+                        nargs='?', default=_DUMMY_VERSION,
```

Comment:
You don't need to use a hacky/dummy version ... you can leave this off and argparse will happily give you a `None` if the argument was missing.